jupyterhub/jupyterhub/overlays/trusted-ca-bundle-path/jupyterhub-singleuser-profiles-configmap.yaml

```diff
@@ -0,0 +1,5 @@
+- op: add
+  path: /data/jupyterhub-singleuser-profiles.yaml/profiles[0]/env/-
```

Comment:
here, I need some input on whether this path reference and operation is good with respect t the original file with an existing env in base.